docs/nima/sse/sse.adoc

```diff
@@ -0,0 +1,223 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Server-Sent Events in Nima
+:description: Helidon Nima SSE Support
+:keywords: helidon, nima, sse
+:feature-name: SSE
+:rootdir: {docdir}/../..
+
+== Contents
+
+- <<Overview, Overview>>
+- <<Server API, Server API>>
```

Comment:
Optional (to include the template format, you could add H3 section headings within the H2)

* * <<Maven Coordinates, Maven Coordinates>>
* * <<Usage, Usage>>